KUVA-418 | Add support for basic internationalization

Jokinen opened the pull request:
## Description

Adds support to three locales by default (some translations pending): `fi`, `sv` and `en`. Translations are implemented with a simple custom hook in order to avoid unduly powerful (and large) dependencies.

## Motivation and Context

All (most) of our application can be used in three different languages. Supporting these out of the box makes the development experience smoother which will likely translate into a better UX for the users as well. For cases where the translations need to be changed or a new locale has to be added, support is also provided.

## How Has This Been Tested?

I added a knob into the storybook environment which allows for manual testing.

## Screenshots (if appropriate):

<img width="1196" alt="Screenshot 2020-07-03 at 16 08 10" src="https://user-images.githubusercontent.com/9090689/86472171-6ac75f00-bd47-11ea-84ef-4a1b403be198.png">

Jokinen committed:
Add support for basic internationalization

tuovinensanttu commented:
Can't test localization on storybook, playground crashes on start.

![Screenshot 2020-07-06 at 9 17 36](https://user-images.githubusercontent.com/25960123/86561563-9c1f7500-bf69-11ea-9fc8-2d0df5e6158f.png)

Jokinen committed:
Fix broken playground

The playground still assumed that the locale could be  an undefined
value and that the FeedbackForm component would replace it with a
sensible default. I changed locale to be a required value--because
setting it by default likely does more harm than good--but forgot to
fix the playground.

Jokinen commented:
Thank you for trying out the playground!

I moved this PR on top of the `develop` branch, which included a force push. I also fixed the problem you were having:
>The playground still assumed that the locale could be  an undefined
>value and that the FeedbackForm component would replace it with a
>sensible default. I changed locale to be a required value--because
>setting it by default likely does more harm than good--but forgot to
>fix the playground.

tuovinensanttu reviewed (approved):
Gave playground another go and everything seems to be working 👍

Jokinen committed:
Fix rollup build
